cmd/signozschemamigrator/schema_migrator/manager.go

```diff
@@ -86,7 +86,6 @@ func NewMigrationManager(opts ...Option) (*MigrationManager, error) {
 		// the default backoff is good enough for our use case
 		// no mutation should be running for more than 15 minutes, if it is, we should fail fast
 		backoff:            backoff.NewExponentialBackOff(),
-		clusterName:        "cluster",
```

Comment:
This will be covered by the `WithClusterName` options provided when `NewMigrationManager` is called with options.